livekit_egress.proto

```diff
@@ -182,6 +189,27 @@ message DirectFileOutput {
   }
 }
 
+enum ImageFileSuffix {
+  IMAGE_SUFFIX_INDEX = 0;
+  IMAGE_SUFFIX_TIMESTAMP = 1;
+}
+
+message ImageOutput {
+  uint32 capture_interval = 1;              // in seconds (required)
+  int32 width = 2;                          // (required)
```

Comment:
could we default to either Track dimensions (for track/participant composite), or a fixed size? would simplify usage.